source/includes/examples/tf-example-autoscaling-main-stagingprod.rst

```diff
@@ -0,0 +1,41 @@
+.. code-block::
+   :copyable: true
+
+   # Create a Project
+   resource "mongodbatlas_project" "atlas-project" {
+     org_id = var.atlas_org_id
+     name = var.atlas_project_name
+   }
+   
+   # Create an Atlas Advanced Cluster 
+   resource "mongodbatlas_advanced_cluster" "atlas-cluster" {
+     project_id = mongodbatlas_project.atlas-project.id
+     name = "ClusterPortalProd"
+     cluster_type = "REPLICASET"
+     mongo_db_major_version = var.mongodb_version
+     replication_specs {
+       region_configs {
+         electable_specs {
+           instance_size = var.cluster_instance_size_name
+           node_count    = 3
+         }
+         analytics_specs {
+           instance_size = var.cluster_instance_size_name
+           node_count    = 1
+         }
+         priority      = 7
+         provider_name = var.cloud_provider
+         region_name   = var.atlas_region
+       }
+     }
+    disk_gb_enabled = var.auto_scaling_disk_gb
+    compute_enabled = var.auto_scaling_compute
+    lifecycle {
```

Comment:
This will fail compilation. This should be:

`ignore_changes = [
replication_specs[0].region_configs[0].electable_specs[0].instance_size,
        replication_specs[0].region_configs[0].electable_specs[0].disk_size_gb,
}`

vars cannot be ignored, so we need to ignore references to actual attributes of the resource